src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
Feel free to remove this file as well.

Reply:
The logo is displayed in the app, so lets not remove it.